benchmarks/mandelbrot/bench.lua

```diff
@@ -0,0 +1,28 @@
+local write, char, unpack = io.write_devnull, string.char, unpack
+
+function run_iter(N)
+  local M, ba, bb, buf = 2/N, 2^(N%8+1)-1, 2^(8-N%8), {}
```

Comment:
Have you double checked the scaling logic here? It's scary.

Reply:
And in the other mandlebrot benchmark.

Reply:
Its the original unmodified scaling logic